cmd/account/mgmt/account-assign.go

```diff
@@ -97,10 +95,16 @@ func (o *accountAssignOptions) run() error {
 		rootID          string
 	)
 
-	if o.payerAccount == "osd-staging-1" {
+	const (
+		osdStaging1    = "osd-staging-1"
+		osdStaging2    = "osd-staging-2"
+		awsAccountName = "AWS_ACCOUNT_NAME"
```

Comment:
Can we make this const name more descriptive? Something like `EnvKeyAWSAccountName`?